angular/projects/global/src/lib/utils/utils.ts

```diff
@@ -22,4 +30,90 @@ export class Utils {
     // trim the last |
     return values.replace(/\|$/, '');
   }
+
+  public formatDate(date) {
+    if (date) {
+      const d = new Date(date);
+      let month = '' + (d.getMonth() + 1);
+      let day = '' + d.getDate();
+      const year = d.getFullYear();
+      if (month.length < 2) {
+        month = '0' + month;
+      }
+      if (day.length < 2) {
+        day = '0' + day;
+      }
+      return [year, month, day].join('');
+    } else {
+      return null;
+    }
+  }
```

Comment:
This seems like the kind of stuff MomentJS is good at.  Or is there a reason to do it manually ourselves?

Reply:
This should be tossed as it's not being used anywhere.